src/components/input-dropdown/input-dropdown.component.html

```diff
@@ -39,6 +39,7 @@
                     [(ngModel)]="filter"
                     (input)="filterChange.emit(filter)"
                     (click)="$event.stopPropagation()"
+                    [attr.aria-required]="required"
```

Comment:
This doesn't represent the value of the control, it's a filter field and should never be marked as required.